### PR TITLE
[Tommy Collison] Redux-friends

### PR DESCRIPTION
**Pull request by tommycollison**
@John-Spraul ⛱ Nobody did, in fact, tell me that life would be this way.

### CODE REVIEW

**Comment by tommycollison** [comment]
`yarn add redux react-redux redux-thunk redux-logger axios` is a fun commit

File: README.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ Topics:
 * optional: `redux-logger` package
 
 ## Project Description
+## ProJohn Spraul
```
Comment (John-Spraul) [inline README.md]: 😎

File: friends/src/App.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+          <p>
+            Edit <code>src/App.js</code> and save to reload.
+          </p>
+          <a
+            className="App-link"
+            href="https://reactjs.org"
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            Learn React
```
Comment (John-Spraul) [inline friends/src/App.js]: This doesn't seem like the `App.js` I'm looking for 🤔

File: friends/src/index.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+import { applyMiddleware, createStore } from '../../../Library/Caches/typescript/3.1/node_modules/redux';
+
+const middleware = applyMiddleware(thunk);
+const store = createStore(asYetUnnamedReducer, middleware)
+
+ReactDOM.render(
+    <Router>
+        <App />
+    </Router>
+    , document.getElementById('root'));
```
Comment (John-Spraul) [inline friends/src/index.js]: have you used your `Provider` anywhere, yet? 🕵️ 🐾

**Pull request closed without merging**